packages/lib/client-only/recipient-initials.ts

```diff
@@ -3,4 +3,4 @@ export const initials = (text: string) =>
     ?.split(' ')
```

Comment:
this is more of a string util, do we have a file for that?

Reply:
Doesn't look like it, what do you think about having a file `packages/lib/universal/recipient-utils.ts` where we dump in any logic related to recipients in?

Reply:
Decided on using `packages/lib/utils/recipient-formatter.ts` lmk if you think there's a better place